Userland/Games/Sudoku/Square.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "Square.h"
+
+Square::Square(int x, int y)
+{
+    m_x = x;
+    m_y = y;
+}
```

Comment:
```suggestion
Square::Square(int x, int y)
    : m_x{x}, m_y{y}
{
}
```

These can go in the constructors init list rather than the body.
(You will have to move m_x, and m_y to be the first members of the class)